.github/workflows/test-n-publish.yml

```diff
@@ -55,7 +57,7 @@ jobs:
       run: |
         mkdir -p ${{ matrix.pkg }}/dist/
         sed -nEe 's/name[ ]*=[ ]*"?([a-z0-9_-]+)"?/\1/p' ${{ matrix.pkg }}/pyproject.toml > pkgname
-        python -m pip download --no-deps --python-version 3.8 \
+        python -m pip download --no-deps --python-version 3.8.1 \
```

Comment:
With `3.8` only here, Pip will skip all package versions that specify minimum Python version 3.8.1. As a result, it will download only very old wheels causing issue #30. Telling Pip to use Python 3.8.1 allows it to consider latest packages.

In the test matrix, we should specify 3.8 only to test with the latest Python. As a side note, we are dropping support for 3.8 in September/October when 3.8 will reach its end-of-support.

Reply:
Okay, I get it, so how `pip download` deals with version vs house the Python setup action does is very different, leading to this problem.  I'll keep this in mind.  Thanks for clarifying :)